gtda/mapper/visualization.py

```diff
@@ -94,13 +95,18 @@ def plot_static_mapper_graph(
     else:
         pipe = pipeline
 
+    if node_color_statistic is not None:
+        _node_color_statistic = node_color_statistic
```

Comment:
I wonder whether this distinction between protected and unprotected variables is still needed? Could we get away with just using `node_color_statistic`?

Reply:
I've always seen this mostly as a way to guide the developer or the person reading the code in general. Technically, I think the code would work even without it in this case (as ``None`` is immutable).

Reply:
Fine with me - let's keep it